Frontend/ui-customer/webpack.android.js

```diff
@@ -27,7 +27,7 @@ module.exports = (env, argv) => {
       contentBase: path.join(__dirname, 'dist'),
       host: "0.0.0.0",
       inline: false,
-      port: 8081
+      port: 8080
```

Comment:
if the dev port is changed to make react access 8081 then can we do this change for react only instead of Presto